README.md

```diff
@@ -75,77 +75,116 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
-- Fixed bug causing multi-monitor spanning errors.
-- Added minimum zone size limit to the settings.
-- Fixed issue where re-opened windows don't appear in previously assigned zone.
-- Fixed excluded apps setting to save on text change instead of when leaving focus.
-- Fixed corrupt/outdated plugins load crash. 
-- Fixed issue with FancyZones not working after computer goes to sleep.
-- Added screen reader confirmation to canvas editor when new zones are added.
+- Adjusted "Save and apply" editor button to adjust with text size for localizations. Thanks @niels9001!
+- Fixed "Create new layout" button visibility when in high contrast mode. Thanks @niels9001!
+- Fixed scaling quirks related to editor UI. Thanks @niels9001!
+- Fixed editor crashing when double clicking the "edit layout" button.
+- Fixed issue with editor crashing immediately after displaying zones.
+- Fixed bug when navigating editor options via keyboard where pressing enter on unselected Canvas option launches Grid editor instead.
+- Fixed issue where FancyZones would not restore Console Applications.
+- Fixed Canvas editor and Grid editor window heights. Thanks @niels9001!
+- Fixed crash due to KERNELBASE.dll.
+- Fixed FancyZone icons to be smoother at higher DPI settings. Thanks @niels9001!
+- Fixed crash when changing between zone layouts.
+- Fixed regression where FancyZones does not resize windows on layout change.
+- Adjusted layout settings to reset shortcut key after canceling changes on a particular layout.
+
+### File Explorer add-ons
+
+- Fixed issue where markdown files were still previewed even when "Enable Markdown" was unselected.
+
+### Image Resizer
+
+- Added warning that GIF files with animations may no correctly resize if the encoding used for the files is incompatible.
 
 ### Keyboard Manager
 
-- Fixed screen reader usage bugs to increase intuitiveness.
+- Improved UI for KBM re-mappings list. Thanks @niels9001!
+
+### PowerRename
+
+- Expanding a plugin option in settings can now be toggled. Thanks @niels9001!
+- Fixed race condition causing PowerRename to crash File Explorer. Thanks @ianjoneill!
 
 ### PowerToys Run
 
-- Fixed crashing bug due to missing image file app.dark.png.
-- Fixed URI plugin bug with handling numeric input. Thanks @davidegiacometti!
-- Improved launch performance of PowerToys run on first call. Thanks @davidegiacometti!
-- Changed URI plugin to launch HTTPS by default instead of HTTP. Thanks @chrisharris333!
-- Added confirmation dialog when system commands are executed from PowerToys Run. Thanks @chrisharris333!
+- Fixed lag caused from PowerToys running in background and invoking Alt-Tab.
+- Resolved file not found exception when loading "System.Windows.Controls.Ribbon".
+- Fixed null reference exception crash.
+- Fixed registry plugin load crash.
+- Fixed unauthorized access exception crash when setting registry keys for the utility.
+- Improved visibility of warnings when plugin properties are collapsed. Thanks @niels9001!
```

Comment:
This specific improvement (issue) also is something related to the new style, so was not visible in the earlier version. Don't think it needs to be mentioned.

Reply:
I agree. This was a temporary bug in the new ux. No need to mention that.